_includes/header.html

```diff
@@ -1,3 +1,21 @@
+<div id="topBanner">
+  <div>
+    <span>🎉 Submit your ideas for a talk at Crossplane Community Day on 5/4/2021 —</span>
```

Comment:
this kubecon is an EU event, so let's consider changing this date to something less ambiguous or confusing for our EU community.  e.g. "May 4th"

Reply:
Good catch, @jbw976 — just force-pushed a more explicit date format